catkit/hub/make_folders_template.py

```diff
@@ -1,97 +1,115 @@
 #!/usr/bin/python
-
+from .tools import check_reaction
 import os
-import sys
 import json
 
-import catkit.hub.ase_tools
-from catkit.hub.tools import extract_atoms, check_reaction
-
 username = os.environ['USER']
 
 
 # ---------publication info------------
 def main(
-        title='Fancy title',  # work title if not yet published
-        authors=['Doe, John', 'Einstein, Albert'],  # name required
+        title,
```

Comment:
Based on the way the documentation is written, many of these arguments should not have default values. This is very likely to produce results you don't want.

Reply:
You mean if users don't update the fields and Albert Einstein stays as an author? :-D I basically wanted to provide an example so that the syntax is clear to users. Not sure how we should do it instead.

Reply:
Yeah, you'll get a lot of Albert papers. These sorts of examples belong in the docstring and many of these defaults should probably be left blank to prevent yourself a headache in the future. Check out the modifications I made to the docstring and make sure they explain things correctly.

This is also an example of the Numpy format which generates our documentation automatically for future reference.

Reply:
Good point. The docstring looks right. Yes- this is the proper way to do it.